azure-pipelines.yml

```diff
@@ -39,11 +39,11 @@ resources:
       name: ISS/metaos-hub-sdk-ios
 
 extends:
-  template: v1/Office.Official.PipelineTemplate.yml@OfficePipelineTemplates
+  template: v1/Office.Unofficial.PipelineTemplate.yml@OfficePipelineTemplates
```

Comment:
Do you suspect the Unofficial template to be a permanent solution for PR pipeline

Reply:
I can't say for sure, I shared this with the team that maintains the SDL tooling but they still don't know what the root cause of the failures is. That being said, this is the correct template for PR builds, so even if it didn't fix the issue we should use the Unofficial template here